user_docs/en/changes.md

```diff
@@ -14,6 +14,8 @@
 ### Bug Fixes
 
 * Fixed support for paragraph mouse text unit in Java applications. (#18231, @hwf1324)
+* Fixed a problem where NVDA sometimes freezes completely when using SAPI5 voices. (#18298, @gexgd0419)
+* Fixed a problem where NVDA fails to start with an SAPI5 Eloquence voice and falls back to OneCore voices. (#18301, @gexgd0419)
 
 ### Changes for Developers
```

Comment:
please add the relevant deprecations

Reply:
ah fair point - can you add them for 2025.3 for now, and we can move them later.
It's possible this gets merged into 2025.3 if it's ready soon enough. 
It's just unlikely / a tight deadline.
We've had 2025.2 blow out a couple of extra weeks than expected which gives this more testing time.